upup/models/cloudup/resources/addons/nodelocaldns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -54,7 +54,7 @@ data:
           force_tcp
         }
         prometheus :9253
-        health {{ KubeDNS.NodeLocalDNS.LocalIP }}:{{ NodeLocalDNSHealthCheck }}
+        health {{ if contains ":" KubeDNS.NodeLocalDNS.LocalIP }}[{{ end }}{{ KubeDNS.NodeLocalDNS.LocalIP }}{{ if contains ":" KubeDNS.NodeLocalDNS.LocalIP }}]{{ end }}:{{ NodeLocalDNSHealthCheck }}
```

Comment:
Maybe we should expose `net.JoinHostPort` to templates. Bound to run into this many times.